framework/include/problems/FEProblemBase.h

```diff
@@ -244,6 +244,9 @@ class FEProblemBase : public SubProblem, public Restartable
                                                   const std::string & var_name) override;
   virtual libMesh::System & getSystem(const std::string & var_name) override;
 
+  // To reference to the RestartableEquationSystems
```

Comment:
```suggestion
  /// Get the RestartableEquationSystems object
```

For doxygen comments for method and variable declaration, we use the standard doxygen format which is `///`